composer.json

```diff
@@ -3,40 +3,43 @@
     "license": "MIT",
     "type": "project",
     "description": "Symfony Demo Application",
-    "minimum-stability": "stable",
-    "prefer-stable": true,
+    "minimum-stability": "beta",
+    "prefer-stable": false,
     "replace": {
         "symfony/polyfill-php70": "*",
-        "symfony/polyfill-php72": "*"
+        "symfony/polyfill-php72": "*",
+        "symfony/polyfill-php73": "*",
+        "symfony/polyfill-php80": "*"
     },
     "require": {
-        "php": "^7.2.9",
+        "php": ">= 7.4",
```

Comment:
⚠️ Note: This is a lie. Not all of our dependencies oficially support PHP 8 yet, so we pretend to pull packages for PHP 7.4.